internal/cloud/mutagen/forward.go

```diff
@@ -0,0 +1,98 @@
+package mutagen
```

Comment:
One thing you'll notice is the function called `mutagenbox.DefaultEnv()`. This contains:

- `MUTAGEN_SSH_PATH`: this points to the SSH shim we built, which checks that the VM is active. If the VM is not active, it will terminate the sync session. We could modify that function to also terminate the port-forward sessions. See [code here](https://github.com/jetpack-io/devbox/blob/77e27103698f4f6fb6ad1872986efb3a3bfb55f4/internal/cloud/openssh/sshshim/mutagen.go#L28-L34).
- `MUTAGEN_DATA_DIRECTORY`: determines which `mutagen daemon` is used. We set this so that there is a devbox-specific mutagen daemon, which works independently of any global mutagen that the user may be using otherwise.